commcare_connect/opportunity/utils/completed_work.py

```diff
@@ -44,9 +44,22 @@ def update_status(completed_works, opportunity_access, compute_payment=True):
         opportunity_access.save()
 
 
-def update_work_payment_date(access: OpportunityAccess):
-    payments = Payment.objects.filter(opportunity_access=access).order_by("date_paid")
-    completed_works = CompletedWork.objects.filter(opportunity_access=access).order_by("status_modified_date")
+def update_work_payment_date(access: OpportunityAccess, payment_model_ref=None, completed_work_model_ref=None):
```

Comment:
For future note, utils like these can stay inside the migration file itself, if they aren't used anywhere else.